trustregistry/registry/registry_actors.py

```diff
@@ -48,7 +50,7 @@ async def update_actor(actor_id: str, actor: Actor, db: Session = Depends(get_db
         actor.id = actor_id
 
     try:
-        update_actor_result = crud.update_actor(db, actor=actor)
+        update_actor_result = crud.update_actor(db_session, actor=actor)
     except crud.ActorDoesNotExistException:
         bound_logger.info("Bad request: Actor not found.")
         raise HTTPException(status_code=404, detail="Actor not found.")
```

Comment:
I think all of these "Actor not found" messages should be more informative, because some we're requesting by id, some by did. So it helps to tell the user "Actor with did {} not found"